src/main/java/pages/BasePage.java

```diff
@@ -144,4 +144,8 @@ public WebElement locateElement(By locator) {
         wait.until(ExpectedConditions.visibilityOfElementLocated(locator));
         return getWebDriverInstance().findElement(locator);
     }
+
+    public boolean isElementPresent(By locator) {
+        return !getWebDriverInstance().findElements(locator).isEmpty();
+    }
```

Comment:
There's a method in BasePage called `isElementDisplayed`, does it not do the exact same thing as this one?